### PR TITLE
Document Imports and Exports

### PR DESCRIPTION
**Pull request by DarvinHarutyunyan**

### CODE REVIEW

File: include/ukv/docs.h [hunk 1]
```diff
@@ -153,8 +153,10 @@ typedef struct ukv_docs_write_t {
     ukv_length_t const* lengths;
     ukv_size_t lengths_stride;
 
-    ukv_bytes_cptr_t const* values;
-    ukv_size_t values_stride;
+    ukv_bytes_cptr_t const* values = NULL;
+    ukv_size_t values_stride = 0;
+
+    ukv_str_view_t id_field = "_id";
```
Comment (ashvardanian) [inline include/ukv/docs.h:159]: Default values aren't C99 compatible.

File: tools/dataset.h [hunk 2]
```diff
@@ -1,100 +0,0 @@
-
-#pragma once
-
-#ifdef __cplusplus
```
Comment (ashvardanian) [inline tools/dataset.h:4]: Why did this file move to src? And why is it marked as deleted instead of moved?

File: benchmarks/tabular_graph.cpp [hunk 3]
```diff
@@ -16,6 +16,202 @@
...
+using namespace unum::ukv;
+
+constexpr size_t max_batch_size_k = 1024 * 1024 * 1024;
+constexpr ukv_str_view_t path_k = "./";
+
+static database_t db;
+
+static std::vector<size_t> source_sizes;
+static std::vector<std::string> source_files;
+std::mutex mtx;
```
Comment (ashvardanian) [inline benchmarks/tabular_graph.cpp:44]: What kind if mutex is this? Is it static? What it does?

**Review by ashvardanian — CHANGES_REQUESTED** [review]

File: benchmarks/tabular_graph.cpp (hunk 3 above)
Comment (Arman-Ghazaryan) [reply benchmarks/tabular_graph.cpp:44]: I'm using ctime function which is not thread safe, in this case I'm use mutex.
I can place this mutex as static variable in function, if it better way of using it.

File: tools/dataset.h (hunk 2 above)
Comment (Arman-Ghazaryan) [reply tools/dataset.h:4]: It's fixed.

File: benchmarks/tabular_graph.cpp (hunk 3 above)
Comment (ashvardanian) [reply benchmarks/tabular_graph.cpp:44]: My point is, having a global variable with name “mtx” is a code-smell. Anything is better.

**Pull request merged**